app/src/main/java/ru/chaichuk/hwapp/MainActivity.kt

```diff
@@ -5,22 +5,17 @@ import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.view.View
 import android.widget.Button
+import ru.chaichuk.hwapp.fragments.MoviesListFragment
 
 class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
 
-        val btnOpenMovies = findViewById<Button>(R.id.buttonMovies);
-        btnOpenMovies.setOnClickListener {
-            startActivity(Intent(this, MoviesActivity::class.java))
-        };
-
-        val btnOpenMovieDetails = findViewById<Button>(R.id.buttonMovieDetails);
-        btnOpenMovieDetails.setOnClickListener {
-            startActivity(Intent(this, MovieDetailsActivity::class.java))
-        };
-
+        supportFragmentManager.beginTransaction()
```

Comment:
you can add root fragment via XML 
[source](https://developer.android.com/guide/fragments/create#add-xml)

Reply:
added MoviesListFragment via <fragment> element in <FrameLayout>. Is that better?

Reply:
added MoviesListFragment via fragment element in FrameLayout. Is that better?

Reply:
surely
don't need to check activity state via saveInstanceState
don't need to think about movieList backStack logic

Reply:
ok